aten/src/ATen/native/cpu/Loops.h

```diff
@@ -42,13 +42,13 @@ namespace at { namespace native { namespace {
 
 using namespace vec256;
 
-template <typename traits, std::size_t... I>
+template <typename traits, std::size_t... INDEX>
```

Comment:
No substantive changes here, I guess?

Reply:
Yes, this was causing a bunch of CI failures. On some machines `std::complex` #defines the letter `I`. If you include `complex.h` you will get build errors on some machines.